dashboards/google-cloud-agent-for-sap/metadata.yaml

```diff
@@ -0,0 +1,13 @@
+sample_dashboards:
+  -
+    category: SAP HANA
+    id: agent-for-sap-hana-overview
+    display_name: Agent for SAP HANA Status Overview
+    description: |-
+      This dashboard is based on [Google Cloud Agent for SAP](https://cloud.google.com/solutions/sap/docs/agent-for-sap/latest/all-guides) and has status overview charts displaying HANA Availability, HANA High Availability, HANA HA Replication, HANA Service Status, HA Cluster - Node State,HA Cluster - Resource State for SAP HANA Instances.
+
+  -
+    category: SAP HANA
+    id: agent-for-sap-hana-raw-metrics
```

Comment:
Similarly, I see `agent-for-sap-hana-raw-metrics` and `agent-for-sap-raw-metrics` - these should match.